libraries/engage/locations/components/FulfillmentSelector/FulfillmentSelectorReserve.jsx

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import { i18n } from '../../../core';
+import { IN_STORE_PICKUP_LABEL, IN_STORE_PICKUP } from '../../constants';
+import { StockInfo } from '../StockInfo';
+import { ChangeLocationButton } from '../ChangeLocationButton';
+import { itemRow, itemColumn } from './FulfillmentSelectorItem.style';
+import { useFulfillmentSelectorState } from './FulfillmentSelector.hooks';
+import { container } from './FulfillmentSelectorReserve.style';
+
+/**
+ * Renders the reservation item label.
+ * @returns {JSX}
+ */
+export function FulfillmentSelectorReserve() {
+  const { location, selectedLocation, handleChange } = useFulfillmentSelectorState();
+  const usedLocation = selectedLocation || location;
+
+  const handleChangeLocation = React.useCallback(() => {
+    handleChange(IN_STORE_PICKUP, true);
+  }, [handleChange]);
+
+  return (
+    <React.Fragment>
+      <div>
+        {i18n.text(IN_STORE_PICKUP_LABEL)}
+      </div>
+      {usedLocation && (
+        <div className={classNames(itemRow.toString(), container.toString())}>
+          <div className={itemColumn}>
+            <div>{usedLocation.name}</div>
+            <ChangeLocationButton onClick={handleChangeLocation} />
+          </div>
+          <div className={itemColumn}>
+            <StockInfo location={usedLocation} />
+          </div>
+        </div>
+      )}
+    </React.Fragment>
```

Comment:
When product is not available at location the error message should be shown `locations.fulfillment.error.reserve`
See removed: libraries/engage/locations/components/FulfillmentSelector/FulfillmentSelectorItemReserve.jsx

Reply:
Also when ROPIS is not available, this one should be shown
![image](https://user-images.githubusercontent.com/18292310/76840000-cde3c300-6836-11ea-8f7f-46daf72700e6.png)

Reply:
Done